src/platform/main.c

```diff
@@ -252,6 +252,8 @@ int main(int argc, char *argv[])
             continue;
         }
 
+        rogue.trueColorMode = false;
```

Comment:
Is this related?

Reply:
the existing BrogueSPEAK implementation manually turns all `trueColorMode` checks to `false` checks, essentially skipping any code that deals with `trueColorMode`. i'm not yet positive the reason for it, i'm guessing it's not related to TTS, maybe just a simplification. this should be conditionally false based on whether this is `BROGUE_SPEECH` mode, i just removed all of my `#ifdef BROGUE_SPEECH` for now because VScode wasn't seeing my `define`s.

Reply:
I think it does that to simply hard-code colour effects off, to increase clarity for partially sighted players. IIRC there are other small visual improvements to that effect in Brogue-SPEAK

Reply:
cool, sounds good to me. if you have suggestions on how to do the checks with the compile flags lmk, my strategy was just like
```c
#ifdef BROGUE_SPEECH
rogue.trueColorMode = false
#endif
```

or whatever.
VScode wasnt able to handle that properly so i dropped it for now, but if there's a better way i'll do that

Reply:
also going to be busy for the next few days, should be able to work a bit more on this over the weekend

Reply:
It would be a shame if the TTS features required a recompile (which using `#define` would imply.)  Maybe we can arrange for it to be {dis,en}abled with a command line flag and/or key binding?

Alternately I'd be interesting in seeing what happens if we just leave true color mode alone; maybe it's fine?

Reply:
> if the TTS features required a recompile

I suppose another way to say this is it would be nice if we could distribute just one binary, and have it serve both users who want TTS, and users who want true color.